FluidNC/include/Driver/fluidnc_uart.h

```diff
@@ -8,7 +8,7 @@ class InputPin;
 void uart_init(uint32_t uart_num);
 void uart_mode(uint32_t uart_num, uint32_t baud, UartData dataBits, UartParity parity, UartStop stopBits);
 bool uart_half_duplex(uint32_t uart_num);
-int  uart_read(uint32_t uart_num, uint8_t* buf, size_t len, uint32_t timeout_ms);
+int  uart_read(uint32_t uart_num, uint8_t* buf, uint32_t len, uint32_t timeout_ms);
```

Comment:
Why uint32_t?  size_t is typically used for len arguments to POSIX read() functions.